src/library_noderawfs.js

```diff
@@ -35,7 +38,7 @@ mergeInto(LibraryManager.library, {
     symlink: function() { fs.symlinkSync.apply(void 0, arguments); },
     rename: function() { fs.renameSync.apply(void 0, arguments); },
     rmdir: function() { fs.rmdirSync.apply(void 0, arguments); },
-    readdir: function() { return fs.readdirSync.apply(void 0, arguments); },
+    readdir: function() { return ['.', '..'].concat(fs.readdirSync.apply(void 0, arguments)); },
```

Comment:
Doesn't readdirSync already return '.' and '..' as part of the result?  If not then this makes sense yes.

Reply:
No `readdirSync` actually does not include `'.'` and `'..'`